config/routes.rb

```diff
@@ -425,7 +426,6 @@ def patch(*) end # Disable PATCH requests
         collection do
           get "check_username"
           get "check_email"
-          get "is_local_username"
```

Comment:
Technically someone could be depending on this API endpoint but we can always remove and see if anyone complains since we don't really have a versioned API. Just want to point out the risk here and acknowledge the course of action we've decided to take.

Reply:
I have checked in our internal logging and nobody uses this via the API.

Reply:
Thanks! That should give us a high level of confidence that no body is using it in the wild as well.